src/main/java/seedu/address/logic/parser/quizparsers/QuizParser.java

```diff
@@ -18,14 +20,17 @@
 
     @Override
     public Command<? super Model> parse(String userInput) throws ParseException {
```

Comment:
Maybe can trim the userInput before parting it, so that "    quiz score flset" would still work.

Another thing to notice is that currently it is using rigorous check, for example quiz(space*2)flset: would not pass. This also happens in the Schedule.
Maybe we can develop a StudyBanana customised tokeniser ourselves to solve these problem.

Reply:
Alright! will take note and update in next commits